modules/mobile/spec/request/claims_decision_request_spec.rb

```diff
@@ -10,10 +10,10 @@
     before do
       token = 'abcdefghijklmnop'
       allow_any_instance_of(BenefitsClaims::Configuration).to receive(:access_token).and_return(token)
-      Flipper.enable(:mobile_lighthouse_claims, user)
+      Flipper.enable(:mobile_lighthouse_request_decision, user)
```

Comment:
This needs updated to work correctly, the way it is here sets it for all users. It likely doesn't matter, but if we are going to pass the user object we should use enable_actor. You can see it in modules/mobile/spec/request/payment_information_request_spec.rb line 43.